hanami-utils.gemspec

```diff
@@ -19,6 +19,7 @@ Gem::Specification.new do |spec|
   spec.required_ruby_version = '>= 2.3.0'
 
   spec.add_dependency 'transproc', '~> 1.0'
+  spec.add_dependency 'wisper', '~> 2.0.0'
```

Comment:
That's a good question. I don't like idea of adding more dependencies than needed, but idea of using Wisper was described here: https://discourse.hanamirb.org/t/hanami-2-0-ideas/306/9
That's why I thought that it's something we want to use for this case.